hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/dn/checksum/TestContainerCommandReconciliation.java

```diff
@@ -84,11 +88,11 @@ public static void init() throws Exception {
     testDir = GenericTestUtils.getTestDir(
         TestContainerCommandReconciliation.class.getSimpleName());
     conf = new OzoneConfiguration();
-    conf.set(HddsConfigKeys.OZONE_METADATA_DIRS, testDir.getAbsolutePath());
     conf.setInt(ScmConfigKeys.OZONE_SCM_PIPELINE_OWNER_CONTAINER_COUNT, 1);
     conf.set(OZONE_METADATA_DIRS, testDir.getAbsolutePath());
-    conf.getBoolean(OZONE_SECURITY_ENABLED_KEY,
-        OZONE_SECURITY_ENABLED_DEFAULT);
+    conf.getBoolean(OZONE_SECURITY_ENABLED_KEY, OZONE_SECURITY_ENABLED_DEFAULT);
```

Comment:
I don't understand why this configuration is fetched.

Reply:
I don't either, looks like it slipped in from a previous PR. I'll remove it.